src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,11 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        User userEmail = userService.findByEmail(email);
```

Comment:
I would say `userEmail` is not correct variable name here. It would be more suitable to do just `user`.